wallet/connect/3rd-party-libraries/wagmi.md

```diff
@@ -15,61 +15,75 @@ the MetaMask browser extension and MetaMask Mobile.
 
 ## Prerequisites
 
-- Ensure you have a basic understanding of Ethereum smart contracts and React Hooks.
-- Set up a project with [Wagmi](https://wagmi.sh/react/getting-started).
-- Create an Infura API key and allowlist to [make read-only requests](../../how-to/make-read-only-requests.md).
+- [Node.js](https://nodejs.org/en/) version 20+
+- [npm](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm/) version 9+
+- A text editor (for example, [VS Code](https://code.visualstudio.com/))
+- The [MetaMask extension](https://metamask.io/) or
+  [MetaMask Flask](/developer-tools/dashboard/get-started/create-api) installed
+- Basic knowledge of [React](https://react.dev/)
 
-## Configure the MetaMask connector
+## Steps
 
-In your Wagmi project, configure the MetaMask connector:
+### 1. Set up the project
 
-```javascript
+If you don't have an existing Wagmi project, set up a new project using Wagmi's `create wagmi` command
+with the `vite-react` template:
+
+```bash
+npm create wagmi@latest  --template vite-react
+```
+
+This prompts you for a project name.
+For example, use `mmsdk-wagmi`.
+Once your project is created, navigate into it and install the node module dependencies:
+
+```bash
+cd mmsdk-wagmi && npm install
+```
+
+### 2. Configure the MetaMask connector
+
+In `wagmi.ts`, configure the MetaMask connector with any [parameters](https://wagmi.sh/core/api/connectors/metaMask#parameters).
```

Comment:
any = the required?

Reply:
These are more like options, so I think "any" is appropriate, but the team prefers using Wagmi's term "parameters."